torch/_inductor/utils.py

```diff
@@ -2247,6 +2247,15 @@ def register_op_dtype_propagation_rules(
     )
 
 
+def upcast_compute_type(dtype: torch.dtype) -> torch.dtype:
+    """Maybe upcast [b]float16 to float32"""
+    if config.triton.codegen_upcast_to_fp32 and (
+        dtype == torch.float16 or dtype == torch.bfloat16
```

Comment:
very nit. Just my personal preference. Up to you.
```suggestion
        dtype in {torch.float16, torch.bfloat16}
```